packages/builders/src/messages/embed/UnsafeEmbed.ts

```diff
@@ -125,8 +125,8 @@ export class UnsafeEmbed implements APIEmbed {
 	 *
 	 * @param fields The fields to add
 	 */
-	public addFields(...fields: APIEmbedField[]): this {
-		this.fields.push(...Embed.normalizeFields(...fields));
+	public addFields(...fields: APIEmbedField[] | APIEmbedField[][]): this {
+		this.fields.push(...Embed.normalizeFields(...fields.flat(2)));
```

Comment:
isn't the `flat`tening already done in [`normalizeFields`](https://github.com/thehairy/discord.js/blob/0de772769646055a72c7f44ae7a2c54fa591b18f/packages/builders/src/messages/embed/UnsafeEmbed.ts#L266)? That said, is `normalizeFields` even needed for `UnsafeEmbed`?